AFL Player Cards_2020-09-30.sql

```diff
@@ -7,7 +7,7 @@
 #
```

Comment:
Is this the version of your database you want to be using? You have two database files?